packages/core/src/StarknetWindowObject.ts

```diff
@@ -1,12 +1,69 @@
-import type { AccountInterface, ProviderInterface } from "starknet"
-import type {
-  AccountInterface as AccountInterfaceV4,
-  ProviderInterface as ProviderInterfaceV4,
-} from "starknet4"
+export enum StarknetChainId {
+  SN_MAIN = "0x534e5f4d41494e",
+  SN_GOERLI = "0x534e5f474f45524c49",
+}
+
+export enum Permission {
+  Accounts = "accounts",
+}
+
+type FELT = string
 
-export type AccountChangeEventHandler = (accounts: string[]) => void
```

Comment:
may i ask, the interface of get-starknet, will be different from snjs?
if this is the case, may be better to follow 100% RPC API spec, i can see the some name and field are missing, e.g INVOKE_TXN_V1

and in additional, when developer using get-startknet to get the last/available wallets, the object itself actually has to follow the same spec with snjs

from a developer prospective, means when the developer using get-startknet-core API, he will using get-startknet own defined interface, when the developer using the wallets object, he has to use snjs interface?